cosmoz-data-nav.js

```diff
@@ -23,6 +23,10 @@ import '@neovici/cosmoz-bottom-bar/cosmoz-bottom-bar-view';
 import { translatable } from '@neovici/cosmoz-i18next';
 import '@neovici/cosmoz-page-router/cosmoz-page-location';
 
+import { hauntedPolymer } from '@neovici/cosmoz-utils';
+
+import { useCache } from './lib/cache.js';
```

Comment:
I think it's a good convention to include `use-` in the filename: `use-cache.js`